Whiskey/Tasks/Invoke-WhiskeyProGetUniversalPackageTask.ps1

```diff
@@ -45,7 +45,10 @@ function Invoke-WhiskeyProGetUniversalPackageTask
     $include = $TaskParameter['Include']
     $exclude = $TaskParameter['Exclude']
     $thirdPartyPath = $TaskParameter['ThirdPartyPath']
-    
+    [int]$compressionLevel = 1
+    if($TaskParameter['CompressionLevel'] -and -not [int]::TryParse($TaskParameter['CompressionLevel'], [ref]$compressionLevel) ){
+        Stop-WhiskeyTask -TaskContext $TaskContext -Message ('"{0}" is not a valid Compression Level. it must be an integer between 0-9.' -f $TaskParameter['CompressionLevel']);

```

Comment:
Update error message to include the property name, e.g. `('Property CompressionLevel: ''{0}'' is not a valid compression level. It must be an integer between 0 and 9.'`.